deploy.sh

```diff
@@ -252,28 +252,28 @@ helm repo update
 # Get this script's path
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-# Generate the scripts paths - make sure these are found
-config_script="${DIR}/create_config.py"
-secret_script="${DIR}/create_secret.py"
-
 # Install the Helm Chart using the configuration files, to deploy both a BinderHub and a JupyterHub:
 echo "--> Generating initial configuration file"
-if [ -z "$DOCKER_ORGANISATION" ] ; then
-  python3 $config_script -id=$DOCKER_USERNAME --prefix=$DOCKER_IMAGE_PREFIX --force
+if [ -z "${DOCKER_ORGANISATION}" ] ; then
+  sed -e "s/<docker-id>/$DOCKER_USERNAME/" \
+  -e "s/<prefix>/$DOCKER_IMAGE_PREFIX/" \
+  ./config-template.yaml > ./config.yaml
 else
-  python3 $config_script -id=$DOCKER_USERNAME --prefix=$DOCKER_IMAGE_PREFIX -org=$DOCKER_ORGANISATION --force
+  sed -e "s/<docker-id>/$DOCKER_ORGANISATION/" \
+  -e "s/<prefix>/$DOCKER_IMAGE_PREFIX/" \
+  ./config-template.yaml > ./config.yaml
 fi
 
 echo "--> Generating initial secrets file"
 
-python3 $secret_script --apiToken=$apiToken \
---secretToken=$secretToken \
--id=$DOCKER_USERNAME \
---password=$DOCKER_PASSWORD \
---force
+sed -e "s/<apiToken>/$apiToken/" \
+-e "s/<secretToken>/$secretToken/" \
+-e "s/<docker-id>/$DOCKER_USERNAME/" \
+-e "s/<password>/$DOCKER_PASSWORD/" \
+./secret-template.yaml > ./secret.yaml
 
-# Format name for kubernetes 
-HELM_BINDERHUB_NAME=$(echo ${BINDERHUB_NAME} | tr -cd '[:alnum:]-.' | tr '[:upper:]' '[:lower:]' | sed -re 's/^([.-]+)//' -re 's/([.-]+)$//' )
+# Format name for kubernetes
+HELM_BINDERHUB_NAME=$(echo ${BINDERHUB_NAME} | tr -cd '[:alnum:]-.' | tr '[:upper:]' '[:lower:]' | sed -e 's/^([.-]+)//' -e 's/([.-]+)$//' )
```

Comment:
Removed the `-r` flag from this line as that is not a valid `sed` option on OSX (meaning `deploy.sh` failed when run locally on a Mac). Does this expression need changing or is it ok?

Reply:
I think the fix here is to change '-r' to '-E'. -r is apparently not posix-compliant, where -E is, and we do need to switch on extended regexes here to make it do what we want. Please could you try -E on your mac and see if it works?